pkg/providers/instance/types.go

```diff
@@ -50,7 +51,8 @@ func NewInstance(out *ec2.Instance) *Instance {
 		SecurityGroupIDs: lo.Map(out.SecurityGroups, func(securitygroup *ec2.GroupIdentifier, _ int) string {
 			return aws.StringValue(securitygroup.GroupId)
 		}),
-		Tags: lo.SliceToMap(out.Tags, func(t *ec2.Tag) (string, string) { return aws.StringValue(t.Key), aws.StringValue(t.Value) }),
+		SubnetID: aws.StringValue(out.SubnetId),
```

Comment:
Do we get any of this information for fleet? Note that there is a `NewInstanceFromFleet` function which grabs info from fleet and puts it into this shared instance object.

Reply:
I don't think we need to propagate it. This is just propagated for tests, since we're not actually doing any ec2 fleet calls in the suite tests.